.github/workflows/downstream.yaml

```diff
@@ -0,0 +1,141 @@
+on:
+  pull_request:
+    types: [ labeled ]
+    branches:
+    - main
+name: downstream
+jobs:
+  dependencies:
+    if: ${{ github.event.label.name == 'downstream-check:run' }}
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        java: [8]
+        repo:
+        - accessapproval
```

Comment:
What do you think a good way to maintain this list in future? (No need to implement in this PR)

Reply:
I am looking into options for phase 2. 
Initial thought, a separate job to read the list and then use those values to run the testing script.
Obstacle - Github doesn't offer any easy ways to pass variables to matrix values.

Reply:
You can setup actions to get that data from previous steps.

Example: https://github.com/googleapis/google-api-java-client-services/blob/main/.github/workflows/codegen.yaml#L19 or https://github.com/googleapis/repo-automation-bots/blob/main/.github/workflows/ci.yaml#L11

Reply:
> `${{fromJson(needs.discovery.outputs.services)}}`

The fist job generates a list and it goes to the definition of the matrix. Nice.